### DISCUSSION TITLE
Failed to install Home brew correctly on mac M1 running sonoma 14.5

### DISCUSSION THREAD

**Discussion by Trakatak** [Homebrew/discussions#5457] [Getting started]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 43eaeca50fe3b6c755f3fc9bc6f43669e0db1039
Last commit: 11 hours ago
Core tap JSON: 24 Jun 19:23 UTC
Core cask tap JSON: 24 Jun 19:23 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 3.3.3 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.6.0 => /usr/bin/curl
macOS: 14.5-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libcpsrt.dylib
felipepiedrahita@DESKTOP-Q2SAAHU AI %
```

### Description of issue

I followed the steps for installation but got this message after running Brew doctor. I updated twice and did the  xcode-select --install . It didn't work and I don't know how to find the files to delete on my mac

**Comment by gromgit** [comment]
> I don't know how to find the files to delete on my Mac

`/usr/local/lib/libcpsrt.dylib` is the file `brew doctor` is talking about. If you find the messaging unclear, please suggest a clearer phrasing.

Also, as `brew doctor` says:

> If you didn't put them there on purpose they could cause problems **when building Homebrew formulae** and may need to be deleted.

Since `brew config` confirms that you installed Homebrew in the standard location, normal installs should be from prebuilt bottles, therefore _not_ requiring building. If you run into issues _running_ stuff you installed with Homebrew, _and_ the error messages you get are related to `libcpsrt.dylib`, _then_ post again. As the very first paragraph in the `brew doctor` output says **in bold**:

> **Please note that these warnings are just used to help the Homebrew maintainers with debugging if you file an issue. If everything you use Homebrew for is working fine: please don't worry or file an issue; just ignore this. Thanks!**
